### ISSUE TITLE
Task: Image Augmentation

### ISSUE THREAD

**Issue by VictoriaParkes** [VictoriaParkes/mildew-detection-in-cherry-leaves#9]
Artificially and temporarily create training images to increase the training image dataset using data augmentation.

- [x] Import Keras ImageDataGenerator
- [x] Initialize ImageDataGenerator
- [x] Augment training image dataset
- [x] Augment validation image dataset
- [x] Augment test image dataset
- [x] Plot augmented training image
- [x] Plot augmented validation and test images
- [x] Save the class indices for both the labels into a pickle file

Acceptance criteria
1. Augmented images for training, validation and test sets.
2. Pickle file containing class indices for both labels.
